weather/script.js

```diff
@@ -0,0 +1,62 @@
+const api = { 
+    key: "", 
+    base: "http://api.openweathermap.org/data/2.5/", 
+    city: "Allentown" 
+} 
+
+let getWeather = fetch(`${api.base}weather?q=${api.city}&units=imperial&APPID=${api.key}`) ;
+    
+getWeather.then((response) => { 
+    return response.json(); 
+})
+.then((resData) => { 
+    let weatherNum = Math.round(`${resData.main.temp}`); 
+
+    
+    let desc = (`${resData.weather[0].main}`);
+
+    if (desc == "Clouds") { 
```

Comment:
change this long `if clause` into a `case switch` statement
more efficient to do this way
see here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch